doc/base/classes.xml

```diff
@@ -26310,8 +26310,10 @@
 </class>
 <class name="Line2D" inherits="Node2D" category="Core">
 	<brief_description>
+		A 2 dimensional line.
```

Comment:
A *two* would be better? I'm not native speaker but I think that for smaller numbers it should be text?

Reply:
Actually, in this case, since it's math related and referring to 2D, it should be "2-dimensional"

Reply:
After talking to KidsCanCode we decided to just use '2D'

Reply:
I think 2D is more clear.  Grammatically, any value under 10 is generally spelled out, but in this case 2D works better.

Reply:
It's fine to use 2, 3, 4, 5 in general for smaller number - the only exception tends to be "one". It's common to write numbers this way when you're blogging. The reason: it's reads faster for most.